ci/azure-setup-test-env.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  # Install Node.js
+  - task: NodeTool@0
+    inputs:
+      versionSpec: "12.0.0"
+    displayName: "Install a fixed version of Node"
+  # Install Go
+  - task: GoTool@0
+    inputs:
+      versionSpec: "1.10"
+    displayName: "Install a fixed version of Go"
+  # Install Python
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: "3.6.8"
```

Comment:
Was there something wrong with Python version 3.7.2? The Rust was failing, not the Python

Reply:
I think some tests where failing because Python v3.7.2 is not available.
```
Version spec 3.7.2 for architecture x64 did not match any version in Agent.ToolsDirectory.
Versions in /opt/hostedtoolcache:

2.7.16 (x64)
3.4.8 (x64)
3.5.7 (x64)
3.6.8 (x64)
3.7.3 (x64)
```

Reply:
Got it ... didn't know what was going on. Thanks for explaining @youssefhabri!